app/pages/events/activity_edit.php

```diff
@@ -48,7 +49,9 @@
 $location_label = $v->text("location_label")->label("Nom du Lieu")->required();
 $location_url = $v->url("location_url")->label("URL du lieu");
 $description = $v->textarea("description")->label("Description de l'activité");
-
+$deadline = $v->date("deadline")
```

Comment:
Should we show a deadline field for activities that are linked to an event? Probably not right? 🤔 
Then I guess we could either not show it at all, or show a disabled field with the event's deadline instead.